contrib/local/setup.sh

```diff
@@ -51,6 +51,9 @@ ls $HOME/$CHAIN_DATA_DIR/config
 
 echo "Update app.toml file"
 sed -i -e 's#keyring-backend = "os"#keyring-backend = "test"#g' $HOME/$CHAIN_DATA_DIR/config/client.toml
```

Comment:
you can just use commands
```
persistenceCore config keyring-backend test
persistenceCore config output json
```
... and so on..

Reply:
ya since we were using sed for replacing, just thought made sense to use sed directly.